File: mlb-teams-service/src/main/java/com/mlb/mlbteamsservice/models/MlbPlayerInfo.java
package com.mlb.mlbteamsservice.models;

public class MlbPlayerInfo {
    private int id;
    private String firstName;
    private String lastName;
    private String picture;
    private String primaryPosition;
    private String batSide;
    private String throwSide;
    private String number;
    private String birthCity;
    private String birthStateProvince;
    private String birthCountry;

    public MlbPlayerInfo(int id, String firstName, String lastName, String picture, String primaryPosition, String batSide, String throwSide, String number, String birthCity, String birthStateProvince, String birthCountry) {
        this.id = id;
        this.firstName = firstName;
        this.lastName = lastName;
        this.picture = picture;
        this.primaryPosition = primaryPosition;
        this.batSide = batSide;
        this.throwSide = throwSide;
        this.number = number;
        this.birthCity = birthCity;
        this.birthStateProvince = birthStateProvince;
        this.birthCountry = birthCountry;
    }

    public int getId() {
        return id;
    }

    public String getFirstName() {
        return firstName;
    }

    public String getLastName() {
        return lastName;
    }

    public String getPicture() {
        return picture;
    }

    public String getPrimaryPosition() {
        return primaryPosition;
    }

    public String getBatSide() {
        return batSide;
    }

    public String getThrowSide() {
        return throwSide;
    }

    public String getNumber() {
        return number;
    }

    public String getBirthCity() {
        return birthCity;
    }

    public String getBirthStateProvince() {
        return birthStateProvince;
    }

    public String getBirthCountry() {
        return birthCountry;
    }
}

File: FrontEnd/src/app/models/mlb-player-info.ts
export class MlbPlayerInfo {
  private _id: number;
  private _firstName: string;
  private _lastName: string;
  private _picture: string;
  private _primaryPosition: string;
  private _batSide: string;
  private _throwSide: string;
  private _number: string;
  private _birthCity: string;
  private _birthStateProvince: string;
  private _birthCountry: string;

  constructor(id: number, firstName: string, lastName: string, picture: string, primaryPosition: string, batSide: string, throwSide: string, number: string, birthCity: string, birthStateProvince: string, birthCountry: string) {
    this._id = id;
    this._firstName = firstName;
    this._lastName = lastName;
    this._picture = picture;
    this._primaryPosition = primaryPosition;
    this._batSide = batSide;
    this._throwSide = throwSide;
    this._number = number;
    this._birthCity = birthCity;
    this._birthStateProvince = birthStateProvince;
    this._birthCountry = birthCountry;
  }

  get id(): number {
    return this._id;
  }

  get firstName(): string {
    return this._firstName;
  }

  get lastName(): string {
    return this._lastName;
  }

  get picture(): string {
    return this._picture;
  }

  get primaryPosition(): string {
    return this._primaryPosition;
  }

  get batSide(): string {
    return this._batSide;
  }

  get throwSide(): string {
    return this._throwSide;
  }

  get number(): string {
    return this._number;
  }

  get birthCity(): string {
    return this._birthCity;
  }

  get birthStateProvince(): string {
    return this._birthStateProvince;
  }

  get birthCountry(): string {
    return this._birthCountry;
  }
}
